Rakefile

```diff
@@ -65,7 +70,8 @@ namespace :build do
     transform = RUBY_PLATFORM =~ /darwin/ ? "-s " : "--transform s"
     transform << "',^,#{tar_basename}/,'"
 
-    `tar -C ../manageiq #{transform} -X #{exclude_file} -hcvzf #{tarball} .`
+    # Everything from */tmp/* should be excluded, except for tmp/cache/sti_loader.yml
+    `tar -C ../manageiq #{transform} --exclude-tag='cache/sti_loader.yml' -X #{exclude_file} -hcvzf #{tarball} .`
```

Comment:
@kbrock I agree, while this is a temp file, it's a different type of temporary file,  we might want to reconsider where this file is stored if it becomes more of a headache if for example, we need to add another temporary file to this directory.